client/package.json

```diff
@@ -13,7 +13,11 @@
     "lint": "eslint src"
   },
   "dependencies": {
-    "apollo-client": "^1.9.3",
+    "apollo-cache-inmemory": "^1.2.1",
+    "apollo-client": "2.3.1",
```

Comment:
you should probably add the caret here so we can do minor upgrades unless there's a particular reason you want to force `2.3.1` only.
```
"apollo-client": "^2.3.1",
```